app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(persons: list) -> list:
+    person_array = []
+    for person in persons:
+        person_array.append(Person(name=person["name"], age=person["age"]))
+    for person in persons:
+        if "wife" in person.keys() and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        elif "husband" in person.keys() and person["husband"] is not None:
+            Person.people[person["name"]].husband \
+                = Person.people[person["husband"]]
```

Comment:
u can create a separate variable with Person.people[person["name"]] to write less code,so u can remove line breaking from 18 line